### PR TITLE
add support in additional shell items

### PR DESCRIPTION
**Pull request by lotansery7**
Add support for the following shell items:
- control_panel_category
- "Control Panel Item
- Users Property View

### CODE REVIEW

File: regipy/plugins/ntuser/shellbags_ntuser.py [hunk 1]
```diff
@@ -107,10 +123,85 @@ def _parse_shell_item_path_segment(self, shell_item):
             else:
                 path_segment = '{{{0:s}}}'.format(shell_item.shell_folder_identifier)
 
+        elif isinstance(shell_item, pyfwsi.users_property_view):
```
Comment (mkorman90) [inline regipy/plugins/ntuser/shellbags_ntuser.py:126]: There is a lot of repeating logic here, can you please move this to a common function?

File: regipy_tests/plugin_tests.py [hunk 2]
```diff
@@ -815,11 +815,13 @@ def test_shellbags_plugin_ntuser(shellbags_ntuser):
              'value_name': '0',
              'node_slot': '11',
              'shell_type': 'Directory',
-             'path': '<UNKNOWN: 0x00>\\<UNKNOWN: 0x00>\\rekall',
+             'path': 'Search Folder\\tmp\\rekall',
```
Comment (mkorman90) [inline regipy_tests/plugin_tests.py:818]: :)

File: requirements.txt [hunk 3]
```diff
@@ -5,4 +5,5 @@ inflection==0.5.1
 pytz
 tabulate==0.8.9
 pytest==7.0.1
-libfwsi-python==20220123
+libfwsi-python==20240315
```
Comment (mkorman90) [inline requirements.txt:8]: This is OK for now, but remember I'm going to move the lib*-python packages to the `full` installation.

**Review by mkorman90 — CHANGES_REQUESTED** [review]

File: regipy/plugins/ntuser/shellbags_ntuser.py (hunk 1 above)
Comment (lotansery7) [reply regipy/plugins/ntuser/shellbags_ntuser.py:139]: Done

**Review by mkorman90 — APPROVED** [review]

**Pull request merged**